test/platform.test.ts

```diff
@@ -20,4 +22,223 @@ suite('Platform Tests', () => {
             done();
         });
     });
+
```

Comment:
if you don't think it will cause any issue then I'm going to keep it just to play with it a little bit more 

---
In reply to: [90504000](https://github.com/Microsoft/vscode-mssql/pull/436#discussion_r90504000) [](ancestors = 90504000)